source/avr/pgmspace.h

```diff
@@ -18,8 +18,23 @@
  * conversion of Arduino specific code.
  */
 
+// Not defined in Windows
+#define PROGMEM
+
 typedef const char* PGM_P;
 typedef const void* PGM_VOID_P;
+typedef void PROGMEM prog_void;
```

Comment:
No, they're not needed. Initially, I wanted to keep the headers as close as possible to the original ones. But, they're too different now. So, I'll remove these.